lib/Mojo/Server/Morbo.pm

```diff
@@ -4,29 +4,20 @@ use Mojo::Base -base;
 # "Linda: With Haley's Comet out of ice, Earth is experiencing the devastating
 #         effects of sudden, intense global warming.
 #  Morbo: Morbo is pleased but sticky."
-use Mojo::File 'path';
+use Mojo::Loader qw(load_class);
 use Mojo::Server::Daemon;
+use Mojo::Util qw(deprecated);
 use POSIX 'WNOHANG';
 
+has backend => sub {
+  my $backend
+    = $ENV{MOJO_MORBO_BACKEND} || 'Mojo::Server::Morbo::Backend::Poll';
```

Comment:
Maybe we can use the same logic here as in `Mojolicious::Plugins::load_plugin()`, so you can write "Mojo::Server::Morbo::Backend::Poll", "poll" or "Poll".